src/AddTodoForm.js

```diff
@@ -1,32 +1,30 @@
-import React, { useState } from "react";
+import React, {useState} from "react";
 import InputWithLabel from "./InputWithLabel";
 
-export default function AddTodoForm({ onAddTodo }) {
-  const [todoTitle, setTodoTitle] = useState("");
+const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = useState('');
 
-//originally you doing this directly in the input like this: 
-//onChange((e)=>setTodoTitle(e.target.value)) which is 
-//totally valid, but now that things are moving around it 
-//probably makes more sense to define and pass this 
-//handler into InputWithLabel instead of the state setter.
-  function handleTitleChange(e) {
-    setTodoTitle(e.target.value);
-  }
+    function handleTitleChange(e) {
+        setTodoTitle(e.target.value);
+    }
 
-  function handleAddTodo(e) {
-    e.preventDefault();
-    onAddTodo({ title: todoTitle, id: Date.now() });
-    setTodoTitle("");
-  }
+    function handleAddTodo(e) {
+        e.preventDefault();
+        if (todoTitle.trim() !== '') {
+            onAddTodo(todoTitle);
+            setTodoTitle("");
+        }
```

Comment:
It's nice to see a check like this since the `required` attribute doesn't account for whitespace, but if you're doing this I'd also suggest some sort of message to the usage when a todo isn't added instead of just nothing happening.

```suggestion
    if (todoTitle.trim().length) { //your check is fine, just showing you another way to checking the same thing here
      onAddTodo(todoTitle);
      setTodoTitle("");
    } else {
      alert("Title is required");
    }

```